sdk/python/README.md

```diff
@@ -68,10 +72,65 @@ Test Status is for branch - **_main_**
 |endpoints|online|[online-endpoints-deploy-mlflow-model-with-script](endpoints/online/mlflow/online-endpoints-deploy-mlflow-model-with-script.ipynb)|Deploy an mlflow model to an online endpoint. This will be a no-code-deployment. It doesn't require scoring script and environment.|[![online-endpoints-deploy-mlflow-model-with-script](https://github.com/Azure/azureml-examples/actions/workflows/sdk-endpoints-online-mlflow-online-endpoints-deploy-mlflow-model-with-script.yml/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/sdk-endpoints-online-mlflow-online-endpoints-deploy-mlflow-model-with-script.yml)|
 |endpoints|online|[online-endpoints-deploy-mlflow-model](endpoints/online/mlflow/online-endpoints-deploy-mlflow-model.ipynb)|Deploy an mlflow model to an online endpoint. This will be a no-code-deployment. It doesn't require scoring script and environment.|[![online-endpoints-deploy-mlflow-model](https://github.com/Azure/azureml-examples/actions/workflows/sdk-endpoints-online-mlflow-online-endpoints-deploy-mlflow-model.yml/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/sdk-endpoints-online-mlflow-online-endpoints-deploy-mlflow-model.yml)|
 |endpoints|online|[online-endpoints-triton](endpoints/online/triton/single-model/online-endpoints-triton.ipynb)|Deploy a custom container as an online endpoint. Use web servers other than the default Python Flask server used by Azure ML without losing the benefits of Azure ML's built-in monitoring, scaling, alerting, and authentication.|[![online-endpoints-triton](https://github.com/Azure/azureml-examples/actions/workflows/sdk-endpoints-online-triton-single-model-online-endpoints-triton.yml/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/sdk-endpoints-online-triton-single-model-online-endpoints-triton.yml)|
+|featurestore_sample|notebooks|[1. Develop a feature set and register with managed feature store](featurestore_sample/notebooks/sdk_and_cli/1. Develop a feature set and register with managed feature store.ipynb)|*no description*|[![1. Develop a feature set and register with managed feature store](https://github.com/Azure/azureml-examples/actions/workflows/sdk-featurestore_sample-notebooks-sdk_and_cli-1. Develop a feature set and register with managed feature store.yml/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/sdk-featurestore_sample-notebooks-sdk_and_cli-1. Develop a feature set and register with managed feature store.yml)|
```

Comment:
how come your PR has feature store notebook changes

Reply:
This file is generated by running the readme.py file ([per the contributing guide instructions](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md#set-up-and-pre-pr)). I guess whoever wrote those notebooks didn't run readme.py before completing their PR